core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7355,15 +7355,15 @@ public String toString() {
 
   public static final PropertyKey DORA_UFS_LIST_STATUS_CACHE_TTL =
       durationBuilder(Name.DORA_UFS_LIST_STATUS_CACHE_TTL)
-          .setDefaultValue("2min")
+          .setDefaultValue("48h")
```

Comment:
I am not sure that directory listings often stay unchanged for as long as 48 hours. In cases that new files are being ingested into the UFS directly and users wants to read fresh data from Alluxio, they will find that the directory listings do not update for a long time.

For example, a table directory has many partitions as subdirectories. If the table is partitioned on a daily basis, then new directories are added daily. The 48h cache may be too aggressive in such case.

Reply:
You concern is reasonable.
For read-only use case, this is fine. But as you explained, in some cases the cache may cause stale metadata.

But you see, this is `default` value. If users know they are going to add more files into directories, they can config a smaller TTL, instead of using the default value.

Reply:
Also, as an enhancement, worker needs to check the option passed in `listStatus(AlluxioURI path, ListStatusPOptions options)`.
This is similar to "Metadata Invalidation by Client" section in  https://github.com/Alluxio/alluxio/pull/17302
The client can ask Alluxio to forcible invalidate the cache.

This will be an optimization.

Reply:
Does creating new file in a directory through Alluxio invalidates the cached listing of that directory? If so, I think as long as a user use Alluxio exclusively for both reads and writes, then a default aggressive cache TTL does not matter.

Reply:
> Does creating new file in a directory through Alluxio invalidates the cached listing of that directory? If so, I think as long as a user use Alluxio exclusively for both reads and writes, then a default aggressive cache TTL does not matter.

No, this is not implemented.
This is a good suggestion. Let's have this optimization as TODO.